examples/core-ping.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "http://mozilla.org/schemas/glean/ping",
+  "ping_info": {
+    "ping_type": "core",
```

Comment:
I thought we were going to change the name to minimize confusion? I'm really liking the `lean` ping sent from glean (plus lean makes sense w.r.t. the size)

Reply:
Sure -- I maybe missed that discussion, but no objections here.